FWCore/Framework/bin/BuildFile.xml

```diff
@@ -59,7 +59,7 @@
   <use name="tbb"/>
   <use name="boost"/>
   <use name="boost_program_options"/>
-  <use name="jemalloc"/>
+  <use name="jemalloc-prof"/>
```

Comment:
I wonder if we should make a new executable (e.g. `cmsRunJEProf`)?

Reply:
OK. `cmsRunJE` is there for historical reasons.